packages/linux-headers-deb/linux-headers-deb.pacscript

```diff
@@ -0,0 +1,7 @@
+name="linux-headers-deb"
+gives="linux-headers-deb"
```

Comment:
gives is what the package name is.
```suggestion
gives="linux-headers"
```

Do the same for every Deb pacscript.

Reply:
Doing that now, will be a moment.